Adds a page template for ads.txt file

craiget opened the pull request:
Adds a page template to be served from smittenkitchen.com/ads.txt. Will also need to create the corresponding "Page" using this template on the live site.

This is for the IAB Ads.txt initiative. Similar to robots.txt, the ads.txt file tells potential advertisers which ad networks are authorized to run on smittenkitchen.com. For more information, please see https://iabtechlab.com/ads-txt/

The key is that this file be served from https://smittenkitchen.com/ads.txt with no additional formatting (similar to a robots.txt file). We may need to make occasional/infrequent updates to this file when an ad network is added or removed from the site.

For an example of other ads.txt files in the wild, please see http://www.adweek.com/ads.txt or http://www.businessinsider.com/ads.txt

pippercameron commented:
I've reverted this since WordPress will rewrite the page url and not allow `.txt`.

I've reworked it and deployed this: https://github.com/a8cteam51/smittenkitchen/commit/cde0c7bf3f25242e5cefbfef566ea97b8c6d1073

craiget commented:
That works for us. Thank you!


On Tue, Oct 24, 2017 at 5:17 PM, Spencer Cameron-Morin <
notifications@github.com> wrote:

> I've reverted this since WordPress will rewrite the page url and not allow
> .txt.
>
> I've reworked it and deployed this: cde0c7b
> <https://github.com/a8cteam51/smittenkitchen/commit/cde0c7bf3f25242e5cefbfef566ea97b8c6d1073>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/a8cteam51/smittenkitchen/pull/154#issuecomment-339134644>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABDwuBl8CZ49aiXrYk40pt10WXV89NQsks5svlP3gaJpZM4P_FVc>
> .
>
